[Update] Quick Start: Ruby and TimescaleDB (#953)
timescale/docs: timescaledb/quick-start/ruby.md

production:
  <<: *default
  database: my_app_db
```

#### Create the database

> ```

Reviewer: ```suggestion
```

Reviewer: Should this be here? I noticed in the docs build we place "Now we can run the following `rake` command..." into the code snippet

 timescaledb | 2.1.1   | public     | Enables scalable inserts and complex queries for time-series data
(2 rows)
```

> <highlight type="NOTE">

Reviewer 2: ```suggestion
<highlight type="note">
```

> After you migrate in the *config/environments/test.rb* file
> add:`config.active_record.verify_foreign_keys_for_fixtures = false` to ensure
> that the tests run succesfully because there are internal foreign keys in
> timescaledb.

Reviewer 2: ```suggestion
To ensure that your tests run successfully, add `config.active_record.verify_foreign_keys_for_fixtures = false` 
to your `config/environments/test.rb` file. Otherwise you get an error because TimescaleDB
uses internal foreign keys.
```
